tsdb/challenges/default.json

```diff
@@ -364,17 +372,18 @@
         {
           "name": "refresh-after-force-merge",
           "operation": "refresh"
-        },
+        }
         {# serverless-post-ingest-sleep-marker-start #}{%- if post_ingest_sleep|default(false) -%}
-        {
+        ,{
           "name": "post-ingest-sleep",
           "operation": {
             "operation-type": "sleep",
             "duration": {{ post_ingest_sleep_duration|default(30) }}
           }
-        },
+        }
         {%- endif -%}{# serverless-post-ingest-sleep-marker-end #}
-        {
+        {% if index_mode | default('time_series') is equalto 'standard' %}
+        ,{
```

Comment:
And here

Reply:
If I do that running the challenge fails with parsing error due to additional commas depending on the if outcome.

Reply:
Surely if both are changed, it should work fine. The ifs don't overlap, and the final item in the list is not dependent on an if statement. I can see that if one of those was changed it would cause issues, but changing both should work